sympy/crypto/crypto.py

```diff
@@ -259,6 +259,55 @@ def decipher_shift(msg, key, symbols=None):
     """
     return encipher_shift(msg, -key, symbols)
 
+def encipher_rot13(msg, symbols=None):
+    """
+    Performs the ROT13 encryption on a given plaintext ```msg```.
+
+    Notes
+    =====
+
+    ROT13 is a substitution cipher which substitutes each letter
+    in the plaintext message for the letter furthest away from it
+    in the English alphabet.
+
+    Equivalently, it is just a Caeser (shift) cipher with a shift
+    key of 13 (midway point of the alphabet).
+
+    See Also
+    ========
+
+    decipher_rot13
+    """
+    return encipher_shift(msg, 13, symbols)
+
+def decipher_rot13(msg, symbols=None):
+    """
+    Performs the ROT13 decryption on a given plaintext ```msg```.
+
+    Notes
+    =====
+
+    ```decipher_rot13``` is equivalent to ```encipher_rot13``` as both
+    ```decipher_shift``` with a key of 13 and ```encipher_shift``` key with a
```

Comment:
These should use double back-ticks (\`\`) to be rendered as code in Sphinx. Current render can be seen at https://docs.sympy.org/dev/modules/crypto.html#sympy.crypto.crypto.encipher_rot13.